### ISSUE TITLE
Is it possible to know when replication of data from Supabase is done?

### ISSUE THREAD

**Issue by Grbson00** [marceljuenemann/rxdb-supabase#23]
Hello guys,
I've having a small issue with users that have slower internet connections, when they are launching my app from a new browser, so where no data is stored in rxdb, where a portion of their documents get loaded immediately, and then the rest of the documents arrive 100 by 100, and I'd really need to know if i can somehow know when all the data has been replicated successfully.

Than you in advance!

**Comment by tsekiguchi** [comment]
I think that this would be linked to the Observable in replication state functions in RxDB? Did you ever get this resolved?

https://rxdb.info/replication.html#rxreplicationstate

**Comment by marceljuenemann** [comment]
Exactly, you should be able to look at the replication state by RxDB
